cmd/auth/main.go

```diff
@@ -89,8 +89,14 @@ func main() {
 	if configuration.ValidHosts == "" {
 		configuration.ValidHosts = configuration.Host
 	}
-	if configuration.Redis.Host == "" {
-		logger.Fatal("REDIS_HOST is not provided")
+	if configuration.Redis.Sentinel.Enabled {
```

Comment:
I think this should be moved to Redis pool factory functions; this validation should be performed whenever Redis config is used.

Reply:
Updated:)